pr2_pick_perception/ObjectDetectionRequest object
---
pr2_pick_perception/ObjectList locations
